src/elements/content-preview/ContentPreview.js

```diff
@@ -355,6 +354,16 @@ class ContentPreview extends React.PureComponent<Props, State> {
         }
     }
 
+    /**
+     * Updates the access token used by preview library
+     *
+     */
+    updateApiToken() {
+        if (this.api) {
+            this.api.setToken(this.props.token);
```

Comment:
Should we always destruct props?

Reply:
I'm not sure that is a requirement when we are just using the value once in a small function

Reply:
yeah in small function destructuring doesn't have any benefits.